### PR TITLE
Added oneDPL samples: gamma-correction, stable_sort_by_key

### PR DESCRIPTION
**Pull request by capatober**
Added oneDPL samples: gamma-correction, stable_sort_by_key.

Signed-off-by: Fedorov, Andrey <andrey.fedorov@intel.com>

### CODE REVIEW

File: Libraries/oneDPL/gamma-correction/sample.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "toolchain": ["dpcpp"],
+  "languages": [{"cpp":{}}],
+  "targetDevice": ["CPU", "GPU"],
+  "os": ["linux", "windows"],
+  "builder": ["ide", "cmake"],
+  "guid": "E249AFB8-6580-4CC3-8EF8-5D33C9A41A2C",
+  "ciTests": {
+    "linux": [
+      {
+        "id": "gamma-correction_lin",
```
Comment (mkitez) [inline Libraries/oneDPL/gamma-correction/sample.json:14]: ID is not required if it's a single test object in the list

File: Libraries/oneDPL/gamma-correction/sample.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          "cd build",
+          "cmake ..",
+          "make",
+          "make run"
+        ]
+      }
+    ],
+    "windows": [
+      {
+        "id": "gamma-correction_win",
```
Comment (mkitez) [inline Libraries/oneDPL/gamma-correction/sample.json:26]: Same as above

**Comment by JoeOster** [comment]
Ci is having an issue  with the installation, please take a look at the below
 
ClCompile:  
C:\Program Files (x86)\Intel\oneAPI\compiler\2021.1-beta08\windows\..\windows\bin\dpcpp-cl.exe /W0 /O2/nologo /D _UNICODE /D UNICODE /Zi /WX- /EHsc /c /I"C:\Program Files(x86)\Intel\oneAPI\tbb\latest\include" /Fo"x64\Release\\" 

src\main.cppsrc\\main.cpp(12,10): : error : 'oneapi/dpl/algorithm' file not found[C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.vcxproj]  
#include <oneapi/dpl/algorithm>
           ^~~~~~~~~~~~~~~~~~~~~~  
1 error generated.
Done Building Project "C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.vcxproj" (Rebuild target(s)) -- FAILED.
Done Building Project "C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.sln" (Rebuild target(s)) -- FAILED.

**Comment by capatober** [comment]
> Ci is having an issue with the installation, please take a look at the below
> 
> ClCompile:
> C:\Program Files (x86)\Intel\oneAPI\compiler\2021.1-beta08\windows..\windows\bin\dpcpp-cl.exe /W0 /O2/nologo /D _UNICODE /D UNICODE /Zi /WX- /EHsc /c /I"C:\Program Files(x86)\Intel\oneAPI\tbb\latest\include" /Fo"x64\Release\"
> 
> src\main.cppsrc\main.cpp(12,10): : error : 'oneapi/dpl/algorithm' file not found[C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.vcxproj]
> #include <oneapi/dpl/algorithm>
> ^~~~~~~~~~~~~~~~~~~~~~
> 1 error generated.
> Done Building Project "C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.vcxproj" (Rebuild target(s)) -- FAILED.
> Done Building Project "C:\BuildAgent\work\cd11826e6db2e255\sample-tester\test-resources\gamma-correction\gamma-correction.sln" (Rebuild target(s)) -- FAILED.

This

**Comment by capatober** [comment]
@JoeOster,
This is not sample's issue, but the package is used for CI testing. Please, update it to the latest oneDPL package

**Comment by JoeOster** [comment]
A new package was created today and will be added to the ci on 7/14. We will rerun it at that time and hopefully, that resolves the issue

**Comment by capatober** [comment]
Got it. Thanks!

**Comment by anjgola** [comment]
The location for these samples is not what is indicated by the spreadsheet. Please discuss with Swapna for right location. I will approve this only when it is matched.

**Comment by anjgola** [comment]
Location is clarified in emails, request is not approved due to windows test failures.

**Comment by capatober** [comment]
As I can see CI tests pass now.

File: Libraries/oneDPL/gamma-correction/License.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Copyright 2019 Intel Corporation
```
Comment (anjgola) [inline Libraries/oneDPL/gamma-correction/License.txt:1]: copyright date.

Comment (capatober) [reply Libraries/oneDPL/gamma-correction/License.txt:1]: Thanks. Should it be 2020 or 2019-2020?

**Pull request merged**